packages/react-router-native/index.tsx

```diff
@@ -81,32 +97,34 @@ if (__DEV__) {
  * A <TouchableHighlight> that navigates to a different URL when touched.
  */
 export function Link({
```

Comment:
Consider using React typing for functional components:
```
export const Link: React.FC<LinkProps> = ({
  onPress, // props are typed here
}) => { }
```
Pros: React.FC contains full signature, including function return values.
Cons: can be used only on variables, not on regular function.
Keep in mind, that we don't lose debugging ability, since React devtools and stack traces will pick name up.